playground/backend/internal/storage/cloud_storage_test.go

```diff
@@ -0,0 +1,175 @@
+package storage
```

Comment:
in general: let's add more negative cases

Reply:
what is the negative?

Reply:
Agree. Negative means when the function fails, e.g. it throws an error which we expect.